### DISCUSSION TITLE
Scroll to entry rather than filter?

### DISCUSSION THREAD

**Discussion by ghost** [datalust/seq-tickets#1137] [Help] [answered]
The filtering in Seq is both fast and powerful. I love it.

Is there a way to FIND an entry, rather than filter on it?

For example, I want to type in a particular log entry and have Seq scroll to it so I can then analyze the preceding and following entries from that point.

Filtering doesn't help me in that particular case.

**Comment by nblumhardt** [comment, marked as answer]
@zmagickap thanks for the note. No, this feature doesn't exist right now, but, if you drop down the "Event" menu on an event of interest, you should see some options like "Seek +/- 5 minutes" etc.:

![image](https://user-images.githubusercontent.com/342712/108449827-df87e300-72af-11eb-97a7-8bff83169ce9.png)


This gets pretty close to the experience you need, though searching-within-results and time navigation are two areas we're interested in improving in the future.
